From 32494b1726a881883cb6d317ab76d1ec3f1fa740 Mon Sep 17 00:00:00 2001
From: Bin Lan <bin.lan.cn@windriver.com>
Date: Wed, 5 Mar 2025 10:51:11 +0800
Subject: [PATCH] tests: set the correct directory of plugin shared library

The plugin shared libraries are put in test-plugins-plugins/.libs when 
building in a host. test-plugins-plugins/.libs is not exist when cross 
build. The plugin shared libraries are put in test-plugins-plugins 
directory when cross build.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Bin Lan <bin.lan.cn@windriver.com>
---
 tests/lib/test-plugins.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/lib/test-plugins.sh b/tests/lib/test-plugins.sh
index f83e42baa..939236393 100755
--- a/tests/lib/test-plugins.sh
+++ b/tests/lib/test-plugins.sh
@@ -14,4 +14,4 @@ fi
 # shellcheck source=../utils/utils.sh
 source "$UTILSSH"
 
-"${BT_TESTS_BUILDDIR}/lib/test-plugins" "${BT_TESTS_BUILDDIR}/lib/test-plugins-plugins/.libs"
+"${BT_TESTS_BUILDDIR}/lib/test-plugins" "${BT_TESTS_BUILDDIR}/lib/test-plugins-plugins"
-- 
2.34.1

